src/App.tsx

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const filteredMoviesList = moviesFromServer.filter(
+    movie => {
```

Comment:
the variable name must be visibleMovies

Reply:
@Serghii29  why it couldn't be 'filteredMovies' ? in checklist there was the example with 'filteredCats'